### PR TITLE
Show stale data warning icon

### PR DESCRIPTION
**Pull request by WilliamBZA**
Shows a staleness warning if required

### CODE REVIEW

**Comment by sergioc** [comment]
I think the easiest thing to do for now is to replace the numeric value next to each graph by `<i class="fa fa-exclamation-triangle danger" aria-hidden="true"></i>`

Not entirely sure if adding the `danger` class will paint the icon in red, but it should be easy in some other ways if that's not the case.

**Comment by sergioc** [comment]
I would also add a danger icon after the endpoint (instance) name to better signal the issue and we need a tooltip for the icons.

**Comment by WilliamBZA** [comment]
The graph is rendered within a `svg`, we can't put HTML elements inside a `svg` except the [defined list](https://developer.mozilla.org/en-US/docs/Web/SVG/Element). So I don't think we can put the icon in place of the numeric value. :sad:

**Comment by sergioc** [comment]
Why render the numeric value inside the SVG instead of directly in HTML?

**Comment by WilliamBZA** [comment]
> Why render the numeric value inside the SVG instead of directly in HTML?

So that it sits next to the line that represents the average

File: src/ServicePulse.Host/app/css/particular.css [hunk 1]
```diff
@@ -1465,4 +1465,15 @@ span.metadata.danger, i.fa.fa-trash-o.danger, sp-moment.danger {
 
 .graph svg {
     width:100%;
+}
+
+.endpointrow {
```
Comment (sergioc) [inline src/ServicePulse.Host/app/css/particular.css]: Can we agree on a convention for naming HTML classes? I've been separating multiple names in class with a hyphen to make it easier to read.

Comment (sergioc) [inline src/ServicePulse.Host/app/css/particular.css]: Bootstrap also separates multiple words with an hyphen.

Comment (WilliamBZA) [inline src/ServicePulse.Host/app/css/particular.css]: I've changed to `endpoint-row`

**Comment by sergioc** [comment]
My last comment about naming classes with multiple words gave one example, but you probably want to apply the convention to all newly introduced classes with multiple names. We can also later do a clean-up PR for this.

**Comment by WilliamBZA** [comment]
Good point @sergioc, but this PR only introduced 1 of those so I agree a later clean-up PR would be better rather than holding this specific one up.

**Pull request merged**